CHANGES.txt

```diff
@@ -65,10 +65,6 @@ Changes
 
 - Added support to change the password of existing users using ``ALTER USER``.
 
-- Added an optimization when UNION ALL is used with ORDER BY. The ORDER BY is
```

Comment:
Why not mention this? I guess because UNION ALL is introduced with this release?

Reply:
Yep, see commit msg